MonoGame.Framework/Graphics/GraphicsDevice.cs

```diff
@@ -267,10 +267,39 @@ private void Setup()
             Dispose(false);
         }
 
+        void ClampMultisampleCountToSupportedByCurrentDevice()

```

Comment:
Ideally this method would be ClampPresentationParameters where we clamp all preferred values every time PresentationParameters are changed. Do we have any method like this?